index.ts

```diff
@@ -113,6 +114,7 @@ export {
   transforms,
   renderers,
   Ast,
+  Node,
```

Comment:
I was afraid

```
export * from './src/types';
```

was going to clash with this (re)export but after some preliminary tests I think it's fine.